wiki/python/ESTIMATOR_GUIDE.md

```diff
@@ -56,13 +56,13 @@ At a high level, all cuML Estimators must:
          super().__init__(handle=handle, verbose=verbose, output_type=output_type)
          ...
    ```
-4. Declare each array-like attribute the new Estimator will compute as a class variable for automatic array type conversion
+4. Declare each array-like attribute the new Estimator will compute as a class variable for automatic array type conversion. An order can be specified to serve as an indicator of the order the array should be in for the C++ algorithms to work.
```

Comment:
This sentence sounds a bit clunky, suggestion:

> Optionally specify the preferred order of the C++ algorithm via the order parameter.

Reply:
Yeah, I think the whole estimator guide could use a bit of a remodel after #4908 goes in. I just created issue #5043 to remind us to tackle that afterward.